scripts/DebugTest.ps1

```diff
@@ -0,0 +1,42 @@
+param
```

Comment:
Remember to put a header at the top of this file like the other scripts.

Reply:
Will add